src/worker.ts

```diff
@@ -36,6 +36,17 @@ export default {
         throw new Error("Invalid settings provided");
       }
 
+      if (!envConfigValidator.test(env)) {
+        const errorDetails: string[] = [];
+        for (const error of envConfigValidator.errors(env)) {
+          errorDetails.push(`${error.path}: ${error.message}`);
+        }
+        return new Response(JSON.stringify({ error: `Bad Request: the environment is invalid. ${errorDetails.join("; ")}` }), {
+          status: 400,
+          headers: { "content-type": "application/json" },
+        });
+      }
```

Comment:
this will just validate it but the transformation won't be done. you need to use `Value.Decode` like above